Detect new brands: MeMobile, VOLIA and Detect device for exists brands

sanchezzzhak opened the pull request.

sanchezzzhak committed:
feat(device) detect new brand Me Mobile: Grace 9
feat(device) detect brand Vivo: iQOO Z7 Pro 5G (I2301), iQOO Z7x 5G (I2216), iQOO Neo 7 (I2214), iQOO Z7 5G (I2213, I2207), iQOO Z7s 5G (I2223)
feat(device) detect brand Dcode: Cygnal 3 Pro (DS-CL3 Pro), Cygnal 3 Lite (DS-CL3 Lite), Bold 3 (DS-BD3)
feat(device) detect brand Infinix: Note 30 5G (X6711)
feat(device) detect brand AllDocube: X (U1005E)
feat(device) detect brand Reeder: M8 Go (2020)

sanchezzzhak committed:
Merge remote-tracking branch 'upstream/master' into pr_device_416

sanchezzzhak committed:
feat(device) detect new brand Sunwell: T95Z Plus
feat(device) detect brand Vivo: Y200 5G (V2307), V29e (V2303)
feat(device) detect brand OPPO: A2 5G (PJG110), Reno 10 Pro+ 5G (PHU110), A1 5G (PHS110), Find N3 Flip (PHT110)
feat(device) detect brand Blackview: N6000, A200 Pro
feat(device) detect brand OnePlus: Ace 2 Pro (PJA110)
feat(device) detect brand Sparx: Neo X
feat(device) detect brand AllDocube: iPlay 50 Mini
feat(device) detect brand Polar: P32L55T2CSM, P55U51T2CSM, P43U51T2SCSM, P42L21T2CSM, P32L32T2CSM, P32L21T2SCSM
feat(device) detect brand PolarLine: 42PL11TC-SM, 65PU51TC-SM, 50PU11TC-SM, 24PL51TC-SM
feat(device) detect brand Nubia: Red Magic Gaming Pad (NP01J)

Simbiat commented on Parser/Device/AbstractDeviceParser.php:
```diff
@@ -992,6 +992,7 @@ abstract class AbstractDeviceParser extends AbstractParser
         'M3'  => 'Mecer',
         'M3M' => 'M3 Mobile',
         '0M'  => 'Mecool',
+        'MEM' => 'Me Mobile',
```
It looks like official name is `MeMobile" without a space. At least the closet to official seems to be their page https://www.facebook.com/memobilepakistan/ (their website is not loading for me). Media seems to be mostly using it with space, but I've found some pages without it, too.

Simbiat replied:
also youtube channel https://www.youtube.com/channel/UC-FA0BPxjtVhkfg0IknOawg

Simbiat replied:
Huh, their official website did load for after like 5 minutes of loading (per page), and I finally got to http://www.memobile.pk/legal/ where they are not using space in the name. I would consider legal name to be the "correct" one.

sanchezzzhak replied:
fix

sanchezzzhak committed:
fix: Sunvell

sanchezzzhak committed:
fix: rename brand Me Mobile to MeMobile

sanchezzzhak committed:
Merge branch 'master' into pr_device_416

sanchezzzhak committed:
Merge remote-tracking branch 'upstream/master' into pr_device_416

sanchezzzhak committed:
Merge branch 'master' into pr_device_416

sanchezzzhak committed:
Merge remote-tracking branch 'upstream/master' into pr_device_416

sanchezzzhak committed:
feat(device) detect new brand VOLIA: VOL001
feat(device) detect brand Hi Nova: 11 (MIZ-BD00)
feat(device) detect brand Nokia: C100 (N156DL)
feat(device) detect brand Huawei: Nova 5i (GLK-LX3)
feat(device) detect brand Realme: 11x 5G (RMX3785)
feat(device) detect brand Infinix: Hot 30 5G (X6832), Zero 5G (X6815D)
feat(device) detect brand Vivo: V29 Lite 5G (V2244), T2x 5G (V2312), Y36 (V2324), Y02t (V2325)

sanchezzzhak committed:
fix: detect Vivo brand for V29 Lite 5G
